Applications/WebPortal/WebApp/library/source/pages/account_page/comment_box.tsx

```diff
@@ -5,13 +5,20 @@ interface Properties {
 
   /** The text to display in the comment box. */
   comment: string;
+
+  /** The callback function that updates the comment. */
+  onInput(value: string): void;
```

Comment:
Callbacks need to be optional. Also we're going to go back to using the old member variable syntax:

```onInput?: ...```